docs/sources/configure/options.md

```diff
@@ -396,6 +396,16 @@ expression.
 If other selectors are specified in the same `services` entry, the processes to be
 selected need to match all the selector properties.
 
+| YAML             | Env var | Type                        | Default |
+| ---------------- | ------- | --------------------------- | ------- |
+| `k8s_pod_labels` | --      | map\[string\]string (regular expression) | (unset) |
+
+This selector property will limit the instrumentation to the applications
+running in the Pods having labels with keys matching the provided value as regular expression.
+
```

Comment:
Since this type of property is new, could you put here a YAML example? Something like:

```yaml
discovery:
  services:
    - k8s_namespace: frontend
      k8s_pod_labels:
        instrument: beyla
```

(or whatever else you consider)

Reply:
updated

Reply:
I pushed another update. The example description had another linting issue